modules/dag-l0/src/main/resources/application.conf

```diff
@@ -0,0 +1,50 @@
+state-after-joining = WaitingForDownload
```

Comment:
Is the plan that only non-consensus breaking settings will be included in the configuration? As a user, knowing which values I should vs. should not change has always seemed like a pain point. though perhaps it is not such a concern in practice?

Reply:
I think everything possible should be moved in here and this `application.conf` would be stored within the app resources, so whenever the end user would like to configure something, you would be able to provide your own application.conf and it will be merged together. I think that's the clue of this ticket to make everything configurable instead of keeping hardcoded things in the code. So answering your question:

> Is the plan that only non-consensus breaking settings will be included in the configuration?

No and I think that's intended